fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java

```diff
@@ -1120,12 +1123,30 @@ public TMasterOpResult forward(TMasterOpRequest params) throws TException {
         LOG.info("receive forwarded stmt {} from FE: {}",
                 params.getStmt_id(), clientAddr != null ? clientAddr.getHostname() : "unknown");
         ConnectContext context = new ConnectContext(null);
-        ConnectProcessor processor = new ConnectProcessor(context);
-        TMasterOpResult result = processor.proxyExecute(params);
-        ConnectContext.remove();
-        return result;
+        final TNetworkAddress feEndpoint = new TNetworkAddress();
+        if (clientAddr != null) {
+            feEndpoint.setHostname(clientAddr.getHostname());
+            // mock fe port
+            feEndpoint.setPort(9030);
```

Comment:
error code？

Reply:
This is expected because each time the rpc gets a different port.
eg: xxxx:12600 xxxx:12611 ... We just use a mock port (due to BDB, only one FE can be deployed per IP, so they don't conflict)